ensime_shared/ensime.py

```diff
@@ -217,12 +213,6 @@ def set_cursor(self, row, col):
         self.log("set_cursor: {}".format((row, col)))
         self.vim.current.window.cursor = (row, col)
 
-    def set_cursor(self, char_counter):
-        """Set cursor at nth char in a buffer."""
-        self.log("set_cursor: {}".format((row, col)))
-        cmd = commands["go_to_char"].format(char_counter + 1)
-        self.vim.command(cmd)
```

Comment:
this is a redefinition of an earlier method and has a syntax error in it, so I assumed the other version was the currently used one.